addons/compat_ibuild/functions/fnc_hasHammer.sqf

```diff
@@ -0,0 +1,30 @@
+#include "..\script_component.hpp"
+/*
+ * Author: TenuredCLOUD
+ * Check if player has a hammer
+ *
+ *
+ * Arguments:
+ * None
+ *
+ * Return Value:
+ * 0: BOOL
+ *
+ * Example:
+ * [] call misery_buildex_fnc_hasHammer;
+ *
+*/
+
+private _hasHammer = false;
```

Comment:
This function doesn't need to exist.

`[[QCLASS(craftingHammer), QCLASS(anvilHammer), "WBK_SmallHammer"]] call EFUNC(common,hasItem);`